.travis.yml

```diff
@@ -7,4 +7,16 @@ env:
   global:
     - COMPOSER_ROOT_VERSION="4.x-dev"
     - REQUIRE_RECIPE="4.x-dev"
+    # Ensure builds default to current stable release.
+    # Run a specialised "preview release job" with GraphQL v4, see below.
+    - REQUIRE_GRAPHQL="3.x-dev"
```

Comment:
Won't need to update .travis.yml since we'll now do this in shared-config https://github.com/silverstripe/silverstripe-travis-shared/pull/4

Reply:
Yep I'll remove that again